load-gen/load-gen-dep.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
```

Comment:
There's already a `load-deployment.yaml` file inside `K8S` folder

Reply:
This is integrated with the scripting to allow for run time configuration. The one under K8s is all hard coded.